lms/services/grouping/service.py

```diff
@@ -150,11 +174,46 @@ def get_course_groupings_for_user(
 
         return query.all()
 
+    def get_sections(self, lti_user, course, grading_student_id=None):
+        if not self.plugin.sections_type:
+            return None
```

Comment:
This wouldn't be required if we had standard non-product types. But seeing as it's here, it's a handy way to skip the entire behavior if it's not supported.